modules/builtin/pulseaudio.go

```diff
@@ -0,0 +1,110 @@
+package builtin
+
+import (
+	"log"
+
+	"fyne.io/fyne"
+	"fyne.io/fyne/layout"
+	"fyne.io/fyne/theme"
+	"fyne.io/fyne/widget"
+	"github.com/mattboll/pulseaudio"
+
+	"fyne.io/fynedesk"
+	wmtheme "fyne.io/fynedesk/theme"
+)
+
+var pulseaudioMeta = fynedesk.ModuleMetadata{
+	Name:        "Pulseaudio",
+	NewInstance: NewPulseaudio,
+}
+
+// Pulseaudio is a progress bar module to modify screen pulseaudio
+type Pulseaudio struct {
+	bar       *widget.ProgressBar
+	client    *pulseaudio.Client
+	soundIcon *widget.Button
+}
+
+// Destroy destroys the module
+func (b *Pulseaudio) Destroy() {
+	b.client.Close()
+}
+
+func (b *Pulseaudio) value() (float32, bool, error) {
```

Comment:
I don't think the bool param is ever used

Reply:
Yes good spot, due to refactor the mute value doesn't come from this function anymore